lib/ABI/Analyses/ConvertToCABIFunctionType.cpp

```diff
@@ -126,10 +126,10 @@ checkVectorRegisterSupport(model::VerifyHelper &VH, const model::Type &Type) {
     revng_abort("Unknown type.");
   }
 
-  if (Result)
+  if (Result) {
     VH.setVerified(&Type);
-
-  VH.verificationCompleted(&Type);
+    VH.verificationCompleted(&Type);
+  }
```

Comment:
What's going on? The commit is not very descriptive.

Reply:
The same thing was silently changed in the `model::Type::size` this is modeled after some time in the past, so this coroutine was silently broken.